app/services/chatbot/chatbot-api.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  IChatbotApiServiceState,
+  IChatbotCommonServiceState,
+  ICustomCommand,
+  IDefaultCommand,
+  IChatbotTimer,
+  IChatAlertsResponse,
+  ICapsProtectionResponse,
+  ISymbolProtectionResponse,
+  ILinkProtectionResponse,
+  IWordProtectionResponse,
+  ChatbotSettingSlugs
+} from './index';
+
+export interface IChatbotServerApi {
+  // state + auth
+  state: IChatbotApiServiceState;
+  logIn(): Promise<any | void>;
+  logOut(): void;
+
+  // get
+  fetchChatbotGlobalEnabled(): Promise<any | void>;
```

Comment:
All of these `any | void` types are effectively meaningless.  In fact the large majority of this API interface doesn't provide any type checking value because all of these functions don't have their actual return types specified.